### DISCUSSION TITLE
Deployment on Testnet

### DISCUSSION THREAD

**Discussion by Patotking12** [Trust-Machines/BNS-V2#74] [General]
Hello everyone! 

We have deployed BNS-V2 on testnet and it is ready to be used and tested. 

For those who already had names on .bns testnet you will see that we have airdropped the equivalent on V2 with the same properties.

The .btc namespace has also been migrated.

Please go ahead and start trying it out, we will be adding more features as days go by, so please if you have any suggestions or ideas, but most importantly, if you find any bugs please report them here.

Any support you may need please reach out

Testnet BNS V2 contract
https://explorer.hiro.so/txid/ST27PT00YS01KBAEEETAH45C1H46C3FMJR31SN2S3.TESTNET-BNS-V2?chain=testnet

Front end to interact with BNS-V2
https://testnet-bns.vercel.app/

**Comment by dartman10** [comment]
Hi @Patotking12 - I keep on getting the error below on register step.  It maybe connected with either zonefile argument or Leather itself?  And is the source code publicly available for https://testnet-bns.vercel.app/ ?  Thanks!

<img width="986" alt="abc" src="https://github.com/user-attachments/assets/39093a2c-202c-4785-8e9d-030aa9b3e4a1">

<img width="826" alt="abcd" src="https://github.com/user-attachments/assets/4b7728d6-c02c-4c2e-a7d1-f096564170c6">

**Reply by southguy** [reply]
Seems TM is Scamming BNS Holders there is lots of domains missed and they left from migration.

100% scam
